### PR TITLE
Skip webserver bootstrap

### PR DESCRIPTION
**Pull request by arvy**
Adapting third-party airflow docker image for our use

### CODE REVIEW

File: publish.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -xe
+
+# Install local teads-central
+docker run --rm -i docker-registry.teads.net/curl:master -sL http://ebzpub.ebuzzing.com/files/teads-central/get.sh | sh - 
+
+HASH=$(./teads-central vars hash)	
+
+./teads-central docker tag-and-push --image airflow-docker --branch-tag  --custom-tags $HASH
```
Comment (gbougeard) [inline publish.sh]: Does airflow have a `--version` arg to fetch its current version?

File: Dockerfile [hunk 2]
```diff
@@ -76,4 +76,5 @@ EXPOSE 8080 5555 8793
 
 USER airflow
 WORKDIR ${AIRFLOW_HOME}
-ENTRYPOINT ["/entrypoint.sh"]
+RUN /entrypoint.sh
+ENTRYPOINT ["airflow"]
```
Comment (bjaglin) [inline Dockerfile]: if the image has an entrypoint, it should be a `tool-` repo (it's "active")

Comment (arvy) [inline Dockerfile]: ..that's right - will rename

File: publish.sh (hunk 1 above)
Comment (arvy) [inline publish.sh]: It does. ..so tag it with version as well?

```
[2017-05-29 10:48:25,919] {__init__.py:57} INFO - Using executor SequentialExecutor
  ____________       _____________
 ____    |__( )_________  __/__  /________      __
____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
 _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
   v1.8.0```

Comment (gbougeard) [inline publish.sh]: Yes, this is usually we do it https://github.com/ebuzzing/tool-curl/blob/master/publish.sh#L9

File: publish.sh [hunk 3]
```diff
@@ -5,6 +5,9 @@ set -xe
...
 
-HASH=$(./teads-central vars hash)	
+IMAGE=$(./teads-central vars image)
+HASH=$(./teads-central vars hash)
+VERSION=$(docker run $IMAGE:$HASH version | tail -1 | sed s/" "//g | sed s/v//g)
 
-./teads-central docker tag-and-push --image airflow-docker --branch-tag  --custom-tags $HASH
\ No newline at end of file
+./teads-central docker tag-and-push --image airflow-docker --branch-tag  \
+  --custom-tags $HASH --custom-tags $VERSION
```
Comment (gbougeard) [inline publish.sh]: Is the `HASH` tag really necessary?

Comment (pdalpra) [inline publish.sh]: good point, I don't think it brings any added value here.

Comment (arvy) [inline publish.sh]: probably not. ..we can remove until the requirement becomes more evident

File: publish.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+set -xe
+
+# Install local teads-central
+docker run --rm -i docker-registry.teads.net/curl:master -sL http://ebzpub.ebuzzing.com/files/teads-central/get.sh | sh - 
+
+IMAGE=$(./teads-central vars image)
+HASH=$(./teads-central vars hash)
+VERSION=$(docker run $IMAGE:$HASH version | tail -1 | sed s/" "//g | sed s/v//g)
+
+./teads-central docker tag-and-push --image $IMAGE --branch-tag --custom-tags $VERSION
```
Comment (arvy) [inline publish.sh]: @gbougeard Is `--image $IMAGE` redundant?  Isn't that the default behaviour if it is not provided?

**Comment by arvy** [comment]
@gbougeard LGTY?

**Pull request merged**